recipe/bld.bat

```diff
@@ -1,12 +1,26 @@
-MOVE bin\* %LIBRARY_BIN%
-MOVE include\* %LIBRARY_INC%
-MOVE jre %LIBRARY_PREFIX%\jre
-MOVE lib\* %LIBRARY_LIB%
-MOVE src.zip %LIBRARY_PREFIX%\jre\src.zip
+setlocal EnableDelayedExpansion
+
+XCOPY bin\* %LIBRARY_BIN% /s /i /y
+if errorlevel 1 exit 1
+
+XCOPY include\* %LIBRARY_INC% /s /i /y
+if errorlevel 1 exit 1
+
+XCOPY lib\* %LIBRARY_LIB% /s /i /y
+if errorlevel 1 exit 1
 
 :: ensure that JAVA_HOME is set correctly
```

Comment:
[This section](https://github.com/hadim/openjdk-feedstock/blob/49f31b49cc1414cba6834c276eacb7aeee787747/recipe/bld.bat#L12-L17)
```bat
:: ensure that JAVA_HOME is set correctly
mkdir %PREFIX%\etc\conda\activate.d
echo set "JAVA_HOME_CONDA_BACKUP=%%JAVA_HOME%%" > "%PREFIX%\etc\conda\activate.d\java_home.bat"
echo set "JAVA_HOME=%%CONDA_PREFIX%%\Library" >> "%PREFIX%\etc\conda\activate.d\java_home.bat"
mkdir %PREFIX%\etc\conda\deactivate.d
echo set "JAVA_HOME=%%JAVA_HOME_CONDA_BACKUP%%" > "%PREFIX%\etc\conda\deactivate.d\java_home.bat"
```

... and [the one](https://github.com/hadim/openjdk-feedstock/blob/49f31b49cc1414cba6834c276eacb7aeee787747/recipe/bld.bat#L19-L26) immediately below it
```bat
:: Copy the [de]activate scripts to %PREFIX%\etc\conda\[de]activate.d.
:: This will allow them to be run on environment activation.
FOR %%F IN (activate deactivate) DO (
    if not exist %PREFIX%\etc\conda\%%F.d MKDIR %PREFIX%\etc\conda\%%F.d
    if errorlevel 1 exit 1
    copy %RECIPE_DIR%\scripts\%%F.bat %PREFIX%\etc\conda\%%F.d\%PKG_NAME%_%%F.bat
    if errorlevel 1 exit 1
)
```
seem to be doing the same thing.

It looks like in the [build.sh](https://github.com/hadim/openjdk-feedstock/blob/49f31b49cc1414cba6834c276eacb7aeee787747/recipe/build.sh#L29-L35) file you actually did just what I said.

Reply:
Ok I thought you were talking about the xcopy commands. Yes I forgot to remove the old lines. Will fix.